### PR TITLE
docs: showcase spacings and border radius

### LINKED ISSUE #159: Build Doku Component - Radius Token Component
Create component for vitepress docs.

Figma:
https://www.figma.com/file/YfEUBOHk4J4nYrk04geswG/onyx---Design-System-Library?type=design&node-id=405%3A556&mode=dev&t=w9zDoyEq59OMyq0r-1

### LINKED ISSUE #160: Bulid Doku Component - Spacing Token Component
Build component for vitepress docs to visualize spacing tokens

Figma:
https://www.figma.com/file/YfEUBOHk4J4nYrk04geswG/onyx---Design-System-Library?type=design&node-id=405%3A649&mode=dev&t=w9zDoyEq59OMyq0r-1

### PR DESCRIPTION
**Pull request by larsrickert**
closes #159, closes #160

## Description

Add documentation page for showcasing spacings and border radius.

## Checklist

- [x] The added / edited code has been documented with [JSDoc](https://jsdoc.app/about-getting-started)
- [x] All changes are documented in the documentation app (folder `apps/docs`)
- [x] If a new component is added, at least one [Playwright screenshot test](https://github.com/SchwarzIT/onyx/actions/workflows/playwright-screenshots.yml) is added
- [x] A changeset is added with `npx changeset add` if your changes should be released as npm package (because they affect the library usage)

### CODE REVIEW

File: apps/docs/src/.vitepress/components/OnyxSpacingTokens.vue [hunk 1]
```diff
@@ -0,0 +1,52 @@
+<script lang="ts" setup>
+import DesignTokenCard from "./DesignTokenCard.vue";
+
```
[the remark this answers is not in the document]
Comment (larsrickert) [reply apps/docs/src/.vitepress/components/OnyxSpacingTokens.vue:3]: Why not :D @jannick-ux What do you think?

File: apps/docs/src/tokens/spacings.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# Spacings
```
[the remark this answers is not in the document]
Comment (larsrickert) [reply apps/docs/src/tokens/spacings.md:1]: I added a short explanation text which also adds some whitespace :)
image

File: apps/docs/src/.vitepress/components/OnyxSpacingTokens.vue (hunk 1 above)
[the remark this answers is not in the document]
Comment (larsrickert) [reply apps/docs/src/.vitepress/components/OnyxSpacingTokens.vue:3]: That was a bug, its fixewd

File: apps/docs/src/.vitepress/components/OnyxBorderRadiusTokens.vue [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+  border-width: 1px 1px 1px 0;
+  border-color: var(--onyx-color-base-warning-500);
+  border-style: solid;
+  background: var(--onyx-color-base-neutral-100);
+
+  &--large {
+    height: 5rem;
+  }
+}
+</style>
```
Comment (BoppLi) [inline apps/docs/src/.vitepress/components/OnyxBorderRadiusTokens.vue:48]:
```suggestion
<template>
  <DesignTokenCard
    v-for="borderRadius in tokens"
    :key="borderRadius"
    :name="borderRadius"
    class="token"
  >
    <figure
      class="token__image"
      :class="{
        'token__image--large': borderRadius.endsWith('-xl') || borderRadius.endsWith('-full'),
      }"
      :style="{
        borderTopRightRadius: `var(--${borderRadius})`,
        borderBottomRightRadius: `var(--${borderRadius})`,
      }"
    ></figure>
  </DesignTokenCard>
</template>

<style lang="scss" scoped>
.token {
  margin-bottom: var(--onyx-spacing-md);

  &__image {
    width: 4rem;
    height: 4rem;
    border-width: 1px 1px 1px 0;
    border-color: var(--onyx-color-base-warning-500);
    border-style: solid;
    background: var(--onyx-color-base-neutral-100);

    &--large {
      height: 5rem;
    }
  }
}
</style>
```

File: apps/docs/src/.vitepress/components/OnyxBorderRadiusTokens.vue [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+  gap: var(--onyx-spacing-md);
+}
+
+.radius {
+  width: 4rem;
+  height: 4rem;
+  border-width: 1px 1px 1px 0;
+  border-color: var(--onyx-color-base-warning-500);
+  border-style: solid;
+  background: var(--onyx-color-base-neutral-100);
```
Comment (BoppLi) [inline apps/docs/src/.vitepress/components/OnyxBorderRadiusTokens.vue:42]: in darkmode, the background color is not really visible :O

image

maybe an adjustment in the design tokens is needed?

File: apps/docs/src/.vitepress/components/OnyxSpacingTokens.vue [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+  "onyx-spacing-md",
+  "onyx-spacing-lg",
+  "onyx-spacing-xl",
+  "onyx-spacing-2xl",
+  "onyx-spacing-3xl",
+];
+</script>
+
+<template>
+  <div class="tokens">
```
Comment (BoppLi) [inline apps/docs/src/.vitepress/components/OnyxSpacingTokens.vue:18]: see flat dom adjustment in the other component :)

**Review by BoppLi — DISMISSED** [review]

File: apps/docs/src/.vitepress/components/OnyxBorderRadiusTokens.vue (hunk 4 above)
Comment (larsrickert) [reply apps/docs/src/.vitepress/components/OnyxBorderRadiusTokens.vue]: #199

**Review by BoppLi — APPROVED** [review]

**Pull request merged**